templates/repo/diff/box.tmpl

```diff
@@ -1,7 +1,8 @@
+{{$showFileTree := (and (not .DiffNotAvailable) (gt .Diff.NumFiles 1))}}
```

Comment:
The double negation is an interesting choice.
Had to think way too hard about it.

Reply:
Yeah, negated vars are bad, but you can read it as "if diff is available and number of files is greater than one".